Tests/SonarScanner.MSBuild.Tasks.UnitTests/GetAnalyzerSettingsTests.cs

```diff
@@ -221,6 +289,19 @@ public void GetAnalyzerSettings_ConfigExists_NewBehaviour_SettingsMerged()
                 "original.should.be.preserved.txt");
         }
 
+
+        [TestMethod]
+        public void ShouldMerge_MissingLanguage_ReturnsFalse()
+        {
+            // Legacy server version - empty language
+            var logger = CheckShouldMerge("7.3.1", "", ignoreExternalIssues: "true", expected: false);
+            logger.AssertInfoMessageExists("Analysis language is not specified");
+
+            // New server version - null language
```

Comment:
Low-level checks for empty and null (although I think the value should never be null as MSBuild will set the task value to _empty_ if the value isn't specified.